sql/core/src/test/scala/org/apache/spark/sql/streaming/continuous/EpochCoordinatorSuite.scala

```diff
@@ -186,6 +192,74 @@ class EpochCoordinatorSuite
     verifyCommitsInOrderOf(List(1, 2, 3, 4, 5))
   }
 
+  test("several epochs, max epoch backlog reached by partitionOffsets") {
+
```

Comment:
nit: there's a bunch of empty lines in these tests for so little code; that makes it weird to read the code.

```
doSomething()
* brain pauses when it sees empty lines *
doSomething()
* another brain pause *
etc etc
```

Reply:
Compacted the tests added by me.